source/sxl_traffic_light_controller.rst

```diff
@@ -1304,516 +2038,950 @@ Commands
+  for the same signal group, you need to use of the other referencing methods.
+- Reference an input by setting 'inputId'. This can be useful if you previously used
+  inputs to activate priority. The input will not be activated, only the priority.
+- Reference a connection by setting 'connectionId'. A connection is a movement from a specific
+  ingoing lane to a specific outgoing lane.
+- Reference an intersection approach by setting 'approachId'.
+- Reference an ingoing lane by setting 'laneInId', and
+  optionally also reference an outgoing lane by setting 'laneOutId'.
+
+Referencing attributes that are not used must be left out, rather than set to null or empty strings.
+This includes:
+
+- signalGroupId
+- inputId
+- connectionId
+- approachId
+- laneInId
+- laneOutId
+
+Referencing attributes are only used when initiating a request. When updating or cancelling the request,
+the request is identified by its requestId, and no referencing attributes are allowed.
+
+You initiate a priority request with type set to 'new'. You must provide a request id that
+uniquely identifies the request on the controller. It can be a randomly generated UUID
+(universally unique identifier), or it can be constructed by combining e.g. a vehicle id
+and some other identifier. When updating or cancelling a request, you must pass the same request id again.
+
+Providing ETA (estimated time of arrival) when initiating a request is optional,
+but can help the controller plan ahead in cases where you're able to send the request before
+the vehicle arrives at the intersection. You're allowed to initiate the request without an ETA
+and provide it in a later request update. But providing the ETA when initiating
+the request is recommended, since it will give the controller more time to plan ahead.
+
+Like ETA, providing a vehicle type is optional, but can help the controller decide how
+to best handle the request.
+
+The priority level provides a way to indicate the relative importance of the request compared
+to other requests. For example, emergency vehicles or delayed buses could be given a higher priority level.
+
+A request with a higher level can override existing priorities with lower levels.
+
+If the ETA changes before the priority is cancelled, or you want to change the priority level, you can send
+another request message with type set to 'update'. The vehicle type cannot be changed.
+
+
+When you send a priority request, it will be processed to decide if it's possible to activate the requested priority.
+
+If the request is accepted, the priority can either be activated immediately, or if another priority is
+currently active, it can be queued for later activation.
+
+If the priority cannot be accepted the request is rejected. Cooldown is a specific type of rejection, which means that
+s similar request has just completed, and some time needs to pass before a similar request can be activated.
```

Comment:
I think cooldown is specifically for 'similar request recently active'.

If another request is currently active it is treated like any other request. This means that it can either:
1. Accept and (possibly) extend any active priorization (if it is a similar priority)
2. Queue (if it is possible)
3. Reject

Reply:
ok then we can leave the text